### ISSUE TITLE
Wasserstein Loss Function

### ISSUE THREAD

**Issue by rnett** [deeplearning4j/deeplearning4j#7376]
Wasserstein loss is being used more in Gans and autoencoders, it seems like it would be a good thing to have.

From Keras, you can implement the Wasserstein Loss as `mean(labels * activations)`.  This seems simple to implement.

**Comment by AlexDBlack** [comment]
https://github.com/deeplearning4j/deeplearning4j/pull/7406
